cfgov/v1/migrations/0015_feedback.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import wagtail.wagtailsnippets.blocks
+import wagtail.wagtailcore.fields
+import v1.models.snippets
+import wagtail.wagtailimages.blocks
+import django.db.models.deletion
+import wagtail.wagtailcore.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0028_merge'),
+        ('v1', '0014_modify_half_blob_labels'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Feedback',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('is_helpful', models.BooleanField()),
+                ('comment', models.TextField(null=True, blank=True)),
+                ('page', models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, to='wagtailcore.Page', null=True)),
+            ],
+        ),
+        migrations.AddField(
```

Comment:
The following steps fail when running on a new local database:

```sh
$ ./drop-db.sh
$ ./create-mysql-db.sh
$ ./initial-data.sh
```

The reason is documented at GHE Platform issue 323. @Scotchester and I hit this last week, and a temporary workaround is to remove the changes introducted to `v1.DemoPage` (don't add a new Streamfield there, which will remove this step from this migration). Pending a more comprehensive solution to this problem, can you remove the changes to the `DemoPage` class, as they don't seem critical/necessary for this PR?